### PR TITLE
Fix type of return value of fused function

### LINKED ISSUE #227: Fused function returns the last argument when the target function returns None
Code:
```
import cupy

@cupy.fuse()
def func(a, b):
    pass

x = cupy.ones((2, 2))
y = cupy.ones((2, 2)) *3
z = func(x, y)
print(z)
```

Results:
```
[[ 3.  3.]
 [ 3.  3.]]
```

### PR DESCRIPTION
**Pull request by asi1024**
Fix #227.

### CODE REVIEW

**Comment by okuta** [comment]
How about adding option to ElementwiseKernel?

**Comment by asi1024** [comment]
The type of the return values can't be determined statically.

**Comment by asi1024** [comment]
Sorry, I accidentally closed it.

File: cupy/core/elementwise.pxi [hunk 1]
```diff
@@ -497,6 +497,9 @@ cdef class ElementwiseKernel:
         options (list): Options passed to the ``nvcc`` command.
         preamble (str): Fragment of the CUDA-C/C++ code that is inserted at the
             top of the cu file.
+        no_return (bool): If ``True``, __call__ returns ``None``.
+        simple_return (bool): If ``False``, __call__ always returns tuple of
```
Comment (okuta) [inline cupy/core/elementwise.pxi]: How about `return_tuple` ?

**Comment by okuta** [comment]
LGTM!

**Pull request merged**